docs/advanced/advanced_examples.rst

```diff
@@ -0,0 +1,27 @@
+.. _advanced-advanced_examples-label:
+
+=================
+Advanced examples
+=================
+
+Setting style
+=============
+
+The default style of ``plothist`` is presentation-ready, with large and readable axis labels and legend. To have a paper-ready plotting style, you can use:
+
+.. code-block:: python
+
+    from plothist import set_paper_style
```

Comment:
paper->publication

Reply:
Actually, we may prefer a `set_style()` taking a string as an argument, to have more options in the future 

* set_style("cyber-criminal")